src/main/java/io/jenkins/plugins/postgresql/jettison/JettisonStaxWriter.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Yikes, so we decided to copy the file?

Reply:
Yeah, can't seem to find a better way since masking didn't work :(